charts/prow/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+Prow is now deployed:
```

Comment:
Helm uses NOTES.txt as a template to output in the terminal after a chart gets installed. See https://github.com/kubernetes/helm/blob/master/docs/charts.md